docs/QSG/setup_suse.md

```diff
@@ -318,11 +319,16 @@ Examples are available on [github](https://github.com/daos-stack/daos/tree/maste
 			log_file: /tmp/daos_engine_0.log
 			env_vars:
 				- CRT_TIMEOUT=30
-			scm_mount: /mnt/daos0
-			scm_class: dcpm
-			scm_list: [/dev/pmem0]
-			bdev_class: nvme
-			bdev_list: ["0000:81:00.0"]  # generate regular nvme.conf
+			storage:
+			-
+				class: dcpm
+				scm_mount: /mnt/pmem0
```

Comment:
/mnt/pmem0 -> /mnt/daos0 to match PR 10582 for release/2.2

Reply:
done

Reply:
done